cukinia

```diff
@@ -681,6 +682,135 @@ _cukinia_systemd_failed()
 	fi
 }
 
+# _cukinia_gpio: Check gpio configuration
+# arg1: (optional) the pins configured in input
+# arg2: (optional) the pins configured in output high
+# arg3: (optional) the pins configured in output low
+# arg4: (optional) the gpiochip used (default:gpiochip0)
+_cukinia_gpio()
```

Comment:
* this function is too big/complicated, we should create at least 2 (at least for the gpiod/sysfs cases) which are selected accordingly
* the UX for this feature looks (too) complicated as well, can you provide an example ?
* keep the coding style the same as the rest of the program